ils_middleware/dags/stanford.py

```diff
@@ -209,7 +239,8 @@ def dag_failure_callback(ctx_dict) -> None:
                     "SIRSI": [
                         "process_symphony.post_new_symphony",
                         "process_symphony.post_overlay_symphony",
-                    ]
+                    ],
+                    "FOLIO": ["process_folio.new-folio-records"],
```

Comment:
Do we not also need an overlay? If folio can handle both cases in one handler, should this have a better name?

Reply:
Done, with @jgreben confirming that the post to the endpoint supports `upsert`, renamed this task.